src/scribe_data/wikidata/query_data.py

```diff
@@ -262,7 +264,39 @@ def query_data(
                 / target_type
                 / f"format_{target_type}.py"
             )
-            os.system(f"python3 {formatting_file_path} --file-path {output_dir}")
+            # Replace the original os.system call with:
+            execute_formatting_script(formatting_file_path, output_dir)
+
+            # os.system(f"python3 {formatting_file_path} --file-path {output_dir}")
+
+
+def execute_formatting_script(formatting_file_path, output_dir):
+    # Determine the root directory of the project
+    project_root = Path(__file__).parent.parent
+
+    if sys.platform.startswith("win"):
+        python_executable = sys.executable
+        pythonpath = str(project_root)
+
+        # Create environment with updated PYTHONPATH
```

Comment:
No stress now, @axif0, but can we get periods at the end of full line comments in the future. Pythonic-wise they should be complete sentences :) I'll fix this locally and send it along shortly!